tests/integration/models/billing.spec.coffee

```diff
@@ -24,35 +24,35 @@ describe 'Billing Model', ->
 		describe 'resin.models.billing.getPlan()', ->
 			it 'should return a free tier billing plan object', ->
 				promise = resin.models.billing.getPlan()
```

Comment:
You can get rid of the `promise =` now too

Reply:
Oh, missed that.